kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -45,6 +45,19 @@ pub enum PropertyClass {
     ///
     /// SPECIAL BEHAVIOR: "Errors" for this type of assertion just mean "reachable" not failure.
     Cover,
+    /// A specialization of the `Cover` class used for collecting coverage information.
```

Comment:
What about calling it CodeCoverage instead?

I would also say something like used for "code coverage instrumentation". I don't think you should mention specialization of "Cover", since they have very different purpose as you mentioned.

Reply:
Makes sense, done!